scripts/linux/agent_diagnostics.sh

```diff
@@ -103,9 +117,17 @@ function info_out() {
   # Write the output.log file.
   SERVICEVERSION=$( cat /opt/jc/version.txt )
   SYSINFO=$( uname -rs )
-  OS=$( grep PRETTY_NAME /etc/os-release | sed 's/\=/:/g' | cut -d':' -f 2 | sed 's/\"//g' )
+  if [[ -f /etc/os-release ]]; then
+    OS=$( grep PRETTY_NAME /etc/os-release | cut -d\" -f2 )
+  else
+    OS=$( grep release /etc/redhat-release )
+  fi
   SERVICE="jcagent"
   STATUS=$( service ${SERVICE} status 2> /dev/null )
+  if [[ -z "${STATUS}" ]]; then
+    STATUS=$( echo "PID  PATH" ; ps ax | grep [j]cagent | awk '{print $1" "$5}')
```

Comment:
What is the purpose of writing `[j]cagent`? The result is the same as having `jcagent`.

Reply:
ps ax | grep jcagent
produces a result that includes the grep jcagent command.

ps ax | grep [j]cagent
produces a result without the grep jcagent command

the end result is not having to deal with the extra line of irrelevant output.

Reply:
Oh, that makes sense. I didn't consider the fact that the `grep` was being applied to `ps` output.